script/Ciel/Day 7/07_01_CIEL07_7.txt

```diff
@@ -397,40 +402,40 @@ The boy watches the Executor reminisce about the past with a cold look in his ey
 [sha:55480e8a88ff6fd527e159b5e2675f6821e2c3ee]{
 -- Page 609, Offset 16272.
 -- 司祭代行はわずかに歯ぎしりをした後、
-The Acting Presbyter briefly grinds his teeth together in frustration, and then――
+The Acting Presbyter briefly grinds his teeth together in frustration, and then―――
 }
 [sha:005882d7a638be84b7f6a2e1d5da6be38aecdf00]{
 -- Page 610, Offset 16273.
 -- 「いい、シラけた。もうテメェとは交渉なんざしてやらねぇ。勝手にひとりでトチ狂ってろ。
 "Whatever, I'm over it.
- Ain't gonna be negotiating with you any longer.
- You can go play around on your own for all I care.
+ Not gonna sit here and bicker with you any longer.
```

Comment:
Feel like negotiations is still appropriate to include. It's quite the difference between it and 'bicker'.